### ISSUE TITLE
Make error due to missing file [highlight_bg.svg]

### ISSUE THREAD

**Issue by glitsj16** [micheleg/dash-to-dock#664]
Recent [commit](https://github.com/micheleg/dash-to-dock/commit/166369cf021076992822353640ba98346225f03e) adds `highlight_bg.svg` to Makefile, but that file does not exist. This causes a make error when building dash-to-dock from git master.

Regards

**Comment by KillWolfVlad** [comment]
`Arch Linux` can't build https://aur.archlinux.org/packages/gnome-shell-extension-dash-to-dock-git package with latest commit 166369cf021076992822353640ba98346225f03e in `master` (but commit 1a27bb4462b71c25239d51d66baf293eb7caa18d works perfect)

```
==> Building and installing package
==> Making package: gnome-shell-extension-dash-to-dock-git 61-1
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Cloning gnome-shell-extension-dash-to-dock git repo...
Cloning into bare repository '/tmp/yaourt-tmp/aur-gnome-shell-extension-dash-to-dock-git/gnome-shell-extension-dash-to-dock'...
remote: Counting objects: 9450, done.
remote: Compressing objects: 100% (76/76), done.
remote: Total 9450 (delta 68), reused 92 (delta 53), pack-reused 9320
Receiving objects: 100% (9450/9450), 6.63 MiB | 5.44 MiB/s, done.
Resolving deltas: 100% (6042/6042), done.
==> Validating source files with md5sums...
    gnome-shell-extension-dash-to-dock ... Skipped
==> Extracting sources...
  -> Creating working copy of  git repo...
Cloning into 'gnome-shell-extension-dash-to-dock'...
done.
==> Starting pkgver()...
==> Updated version: gnome-shell-extension-dash-to-dock-git 62.r17.g166369c-1
==> Starting build()...
glib-compile-schemas ./schemas/
msgfmt -c po/ru.po -o po/ru.mo
msgfmt -c po/pl.po -o po/pl.mo
msgfmt -c po/ja.po -o po/ja.mo
msgfmt -c po/tr.po -o po/tr.mo
po/tr.po:8: warning: header field 'Language-Team' missing in header
msgfmt -c po/sr.po -o po/sr.mo
msgfmt -c po/id.po -o po/id.mo
msgfmt -c po/pt.po -o po/pt.mo
msgfmt -c po/es.po -o po/es.mo
msgfmt -c po/sr@latin.po -o po/sr@latin.mo
msgfmt -c po/ar.po -o po/ar.mo
msgfmt -c po/sv.po -o po/sv.mo
msgfmt -c po/fr.po -o po/fr.mo
msgfmt -c po/hu.po -o po/hu.mo
msgfmt -c po/it.po -o po/it.mo
msgfmt -c po/cs.po -o po/cs.mo
msgfmt -c po/pt_BR.po -o po/pt_BR.mo
msgfmt -c po/sk.po -o po/sk.mo
msgfmt -c po/nl.po -o po/nl.mo
msgfmt -c po/zh_CN.po -o po/zh_CN.mo
msgfmt -c po/de.po -o po/de.mo
msgfmt -c po/zh_TW.po -o po/zh_TW.mo
msgfmt -c po/el.po -o po/el.mo
rm -fR ./_build
mkdir -p _build
cp extension.js stylesheet.css metadata.json COPYING README.md convenience.js dash.js docking.js appIcons.js appIconIndicators.js launcherAPI.js windowPreview.js intellihide.js prefs.js theming.js utils.js Settings.ui _build
mkdir -p _build/media
cd media ; cp logo.svg glossy.svg highlight_bg.svg highlight_stacked_bg.svg highlight_stacked_bg_h.svg ../_build/media/
cp: cannot stat 'highlight_bg.svg': No such file or directory
make: *** [Makefile:83: _build] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
==> ERROR: Makepkg was unable to build gnome-shell-extension-dash-to-dock-git.
==> Restart building gnome-shell-extension-dash-to-dock-git ? [y/N]
```

**Comment by micheleg** [comment]
Thank you! It should be fixed now by  e27c8cc.
